core/src/main/java/org/apache/iceberg/hadoop/HadoopCatalog.java

```diff
@@ -51,13 +54,13 @@ public HadoopCatalog(Configuration conf) {
       }
       this.hdfsRoot = hdfsRoot + "/" + ICEBERG_HADOOP_WAREHOUSE_BASE;
```

Comment:
this was slightly confusing, instead of `hdfsRoot` we should name this variable `warehousePath` or something similar